src/components/admin/branchSettings/BranchFormModal.tsx

```diff
@@ -180,6 +188,25 @@ const BranchFormModal: React.FC<Props> = props => {
                   <Checkbox {...register("secret")}>Secret</Checkbox>
                 </FormControl>
               )}
+              {branchType === "APPLICATION" && secret && (
+                <Popover>
+                  <PopoverTrigger>
+                    <Button>View QR Code</Button>
+                  </PopoverTrigger>
+                  <PopoverContent>
+                    <PopoverArrow />
+                    <PopoverCloseButton />
+                    <PopoverHeader>QR Code</PopoverHeader>
+                    <PopoverBody>
+                      <QRCode
+                        size={256}
+                        style={{ height: "auto", maxWidth: "100%", width: "100%" }}
+                        value={apiUrl(Service.REGISTRATION, `/${hexathonId}/start-application/${props.defaultValues.id}`)}
+                      />
+                    </PopoverBody>
```

Comment:
Can you include more instructions at the top or somewhere for organizers? like telling them to have a participant scan it and like what it will do